features/other_a/invite_request.feature

```diff
@@ -62,16 +62,14 @@ Feature: Invite requests
       And I should not see "Sorry, you have no unsent invitations right now."
       And I should see "You have 2 open invitations and 0 that have been sent but not yet used."
 
-  Scenario: User can fail to send an invitation to an invalid email address
+  Scenario: User can see an error after trying to invide an invalid email address
```

Comment:
Oops. I think I have some more things to fix anyway, and will fix the typo along them later.